test/integration/golden_response/staging/get_statvar_group/uk.json

```diff
@@ -1036,117 +1021,38 @@
-          "specializedEntity": "Amount_Remittance_TransferType"
-        },
-        {
-          "id": "dc/g/Amount_Stock",
-          "specializedEntity": "Amount_Stock"
-        },
-        {
-          "id": "dc/g/CampaignExtent_PublicInformationCampaign_Category_Organizer_SpatialCoverage",
-          "specializedEntity": "CampaignExtent_PublicInformationCampaign_Category_Organizer_SpatialCoverage"
-        },
-        {
-          "id": "dc/g/Count_BirthEvent",
-          "specializedEntity": "Count_BirthEvent"
-        },
-        {
-          "id": "dc/g/Count_CycloneEvent",
-          "specializedEntity": "Count_CycloneEvent"
-        },
-        {
-          "id": "dc/g/Count_EarthquakeEvent",
-          "specializedEntity": "Count_EarthquakeEvent"
-        },
-        {
-          "id": "dc/g/Count_Product_ProductType",
-          "specializedEntity": "Count_Product_ProductType"
-        },
-        {
-          "id": "dc/g/Covid19ContainmentAndHealthIndex_Legislation_Category_LegislationPassedBy",
-          "specializedEntity": "Covid19ContainmentAndHealthIndex_Legislation_Category_LegislationPassedBy"
-        },
-        {
-          "id": "dc/g/Covid19EconomicSupportIndex_Legislation_Category_LegislationPassedBy",
-          "specializedEntity": "Covid19EconomicSupportIndex_Legislation_Category_LegislationPassedBy"
-        },
-        {
-          "id": "dc/g/Covid19MobilityTrend_PlaceVisitEvent_PlaceCategory",
-          "specializedEntity": "Covid19MobilityTrend_PlaceVisitEvent_PlaceCategory"
-        },
-        {
-          "id": "dc/g/Covid19ResponseIndex_Legislation_Category_LegislationPassedBy",
-          "specializedEntity": "Covid19ResponseIndex_Legislation_Category_LegislationPassedBy"
-        },
-        {
-          "id": "dc/g/Covid19StringencyIndex_Legislation_Category_LegislationPassedBy",
-          "specializedEntity": "Covid19StringencyIndex_Legislation_Category_LegislationPassedBy"
-        },
-        {
-          "id": "dc/g/CumulativeCount_MedicalTest_MedicalCondition",
-          "specializedEntity": "CumulativeCount_MedicalTest_MedicalCondition"
-        },
-        {
-          "id": "dc/g/CumulativeCount_Vaccine_MedicalCondition_VaccineStatus",
-          "specializedEntity": "CumulativeCount_Vaccine_MedicalCondition_VaccineStatus"
-        },
-        {
-          "id": "dc/g/IncrementalCount_MedicalTest_MedicalCondition",
-          "specializedEntity": "IncrementalCount_MedicalTest_MedicalCondition"
-        },
+          "id": "dc/g/Amount_EconomicActivity_ActivitySource",
+          "specializedEntity": "Amount_EconomicActivity_ActivitySource"
+        }
+      ]
+    },
+    "dc/g/Employment": {
+      "absoluteName": "Employment",
+      "childStatVarGroups": [
         {
-          "id": "dc/g/IncrementalCount_Vaccine_MedicalCondition_VaccineStatus",
-          "specializedEntity": "IncrementalCount_Vaccine_MedicalCondition_VaccineStatus"
+          "id": "dc/g/Count_Person",
```

Comment:
@pradh dc/g/Count_Person is child of dc/g/Employment, is this expected?

Reply:
Yes, because employment is still a dependent PV. I'll need to do a pass over the configs and update that.